architecture/additional_concepts/authorization.adoc

```diff
@@ -300,7 +300,7 @@ endif::[]
 ifdef::openshift-dedicated[]
 . Running of privileged containers.
 endif::[]
-ifdef::openshift-origin,openshift-origin,openshift-enterprise,openshift-dedicated[]
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
```

Comment:
The 3 new ifdefs in this Arch topic can also include `openshift-online`, I think.